scripts/deploy/bump-version.js

```diff
@@ -32,9 +32,14 @@ async function checkoutLatestMaster() {
   await exec('git pull origin master');
 }
 
+/**
+ * @param {'pre' | 'graduate'} versionType
+ */
 async function bumpVersionAndPush(versionType) {
   try {
-    await exec(`npm run version:${versionType} -- --yes`);
+    const message = '[version bump]: publish %s';
```

Comment:
This new commit message matches both the [Lerna documentation](https://github.com/lerna/lerna/tree/main/commands/version#user-content---message-msg:~:text=lerna%20version%20%2Dm%20%22chore(release)%3A%20publish%20%25s%22) and the [admin-ui commit message](https://github.com/coveo/admin-ui/commit/84d60fde25b3c03c1544480b3fa0cca1622cd679#repo-content-pjax-container:~:text=%5Bversion%20bump%5D%3A%20publish%20%25s).